qemu/tests/qemu_guest_agent_snapshot.py

```diff
@@ -41,18 +43,38 @@ def _action_before_fsfreeze(self, *args):
             dict(verbose=True, timeout=copy_timeout))
         self.bg.start()
 
+    def check_snapshot(self):
+        """
+        Check whether the snapshot is created successfully.
+        """
+        snapshot_info = str(self.vm.monitor.info("block"))
+        snapshot_node_name = self.params.get("snapshot_node_name")
+        if self.params.get("snapshot_file") not in snapshot_info:
+            logging.error(snapshot_info)
+            raise exceptions.TestFail("Snapshot doesn't exist")
+        if snapshot_node_name:
+            match_string = "u?'node-name': u?'%s'" % snapshot_node_name
+            if not re.search(match_string, snapshot_info):
+                logging.error(snapshot_info)
+                raise exceptions.TestFail("Can not find node name %s of"
```

Comment:
the same  as line53-54

Reply:
any update?

Reply:
updated to elf.test.fail("Can not find node name %s of" " snapshot in block info %s" % (snapshot_node_name, snapshot_info))